File: src/Z.Core/System.Double/System.Math/Double.Sqrt.cs
// Description: C# Extension Methods Library to enhances the .NET Framework by adding hundreds of new methods. It drastically increases developers productivity and code readability. Support C# and VB.NET
// Website & Documentation: https://github.com/zzzprojects/Z.ExtensionMethods
// Forum: https://github.com/zzzprojects/Z.ExtensionMethods/issues
// License: https://github.com/zzzprojects/Z.ExtensionMethods/blob/master/LICENSE
// More projects: http://www.zzzprojects.com/
// Copyright © ZZZ Projects Inc. 2014 - 2016. All rights reserved.
using System;

public static partial class Extensions
{
    /// <summary>
    ///     Returns the square root of a specified number.
    /// </summary>
    /// <param name="d">The number whose square root is to be found.</param>
    /// <returns>
    ///     One of the values in the following table.  parameter Return value Zero or positive The positive square root
    ///     of . Negative Equals Equals.
    /// </returns>
    public static Double Sqrt(this Double d)
    {
        return Math.Sqrt(d);
    }
}

File: src (VB.NET)/Z.ExtensionMethods.VB/Z.Core/System.Double/System.Math/Double.Sqrt.vb

' Copyright (c) 2015 ZZZ Projects. All rights reserved
' Licensed under MIT License (MIT) (https://github.com/zzzprojects/Z.ExtensionMethods)
' Website: http://www.zzzprojects.com/
' Feedback / Feature Requests / Issues : http://zzzprojects.uservoice.com/forums/283927
' All ZZZ Projects products: Entity Framework Extensions / Bulk Operations / Extension Methods /Icon Library


Public Module Extensions_225

	''' <summary>
	'''     Returns the square root of a specified number.
	''' </summary>
	''' <param name="d">The number whose square root is to be found.</param>
	''' <returns>
	'''     One of the values in the following table.  parameter Return value Zero or positive The positive square root
	'''     of . Negative Equals Equals.
	''' </returns>
	<System.Runtime.CompilerServices.Extension> _
	Public Function Sqrt(d As [Double]) As [Double]
		Return Math.Sqrt(d)
	End Function
End Module


